spec/prawn/markup/showcase_spec.rb

```diff
@@ -19,7 +19,8 @@
       iframe: {
         placeholder: ->(src) { "Embedded content: #{src}" }
       },
-      input: { symbol_font: 'DejaVu', symbol_font_size: 16 }
+      input: { symbol_font: 'DejaVu', symbol_font_size: 16 },
+      link: { color: { rgb: "0000FF" }, underline: true }
```

Comment:
The showcase spec is ok, but specific specs in `text_spec.rb` for RGB and CYMK are also required.

Reply:
Based on previous comments, I assume that color should be either a string (for RGB hex) or an array (for CMYK format), rather than a hash as stated in the spec. Is that correct?

Reply:
Exactly, see e.g. https://prawnpdf.org/docs/prawn/2.5.0/Prawn/Graphics/Color.html#stroke_color-instance_method